_data/simple-icons.json

```
@@ -10572,6 +10572,11 @@
             "hex": "1F6B75",
             "source": "https://github.com/openlayers/openlayers.github.io/blob/5b93e18b8d302eb49a812fb96abb529895ceb7a2/assets/logo.svg"
         },
+        {
+            "title": "openmediavault",
+            "hex": "5DACDF",
+            "source": "https://www.openmediavault.org"
```

Also, the source will need to be updated.
```suggestion
            "source": "https://github.com/openmediavault/openmediavault/blob/12f8ef70f19f967733b744d6fb6156a4181f1ddc/deb/openmediavault/workbench/src/favicon.svg"
```